x-pack/plugins/infra/public/pages/metrics/metric_detail/components/time_controls.test.tsx

```diff
@@ -47,10 +47,7 @@ describe('MetricsTimeControls', () => {
       .find('[data-test-subj="superDatePickerToggleQuickMenuButton"]')
       .first()
       .simulate('click');
-    component
-      .find('[data-test-subj="superDatePickerCommonlyUsed_Today"]')
-      .first()
-      .simulate('click');
+    component.find('[data-test-subj="superDatePickerCommonlyUsed_Today"]').last().simulate('click');
```

Comment:
Can you help me understand why this changed?

Reply:
@breehall will know

Reply:
This appears to have been introduced by the Emotion team for compatibility with React 18 - https://github.com/emotion-js/emotion/issues/2553#issuecomment-970793818

Reply:
TIL, thank you very much for the explanation!